src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
```

Comment:
The divs that are wrapping up Link component are redundant! If you are using them for styling purpose, Link component can also takes in css className as a props instead.